src/tests/category.service.test.ts

```diff
@@ -0,0 +1,266 @@
+import 'reflect-metadata'
```

Comment:
Are all of these tests being triggered per commit? How is this working? If not - please use them for each commit or PR. 
If they are obsolete - delete them

Reply:
These are two types of tests one per push and one per PR against the main. I've updated the workflow now to include the PR for the dev branch.
Thanks for the review.